### DISCUSSION TITLE
Deploy specific Magento Theme with MAD-I-T/magento-actions

### DISCUSSION THREAD

**Discussion by IvanVizcaino** [MAD-I-T/magento-actions#51] [General]
Hello, We are trying to migrate my deploy system to github Actions. Is possible to specify the themes to deploy static-content with this action? I only see the de Luma and Blanck themes generated.

Thanks

**Comment by seyuf** [comment]
Hi @IvanVizcaino ,

The action builds all the available themes/areas, though you can limit the process to specific locales  [see here](https://forum.madit.fr/t/build-magento-from-github-actions-static-deploy-with-multiple-languages/25/2?u=madit).
As of today, we don't see the necessity to introduce theme separation logic, but feel free to open an issue regarding your specific use case. Or if you think there is a bug somewhere?

Best

**Reply by seyuf** [reply]
https://github.com/MAD-I-T/magento-actions/commit/e4457121743170e912246588236ec89da4da2174

**Reply by IvanVizcaino** [reply]
Thank you so much!

**Reply by seyuf** [reply]
You're welcome.
You'll have to use the @master for now. Before, new release probably by the end of the week.
https://forum.madit.fr/t/limit-magento-static-content-build-to-specific-theme-s-on-magento-actions/97
